### DISCUSSION TITLE
GUI 和  打包

### DISCUSSION THREAD

**Discussion by xx025** [xx025/YanX#27] [Ideas]
2023年8月5日

当前使用 flaskwebgui 和 fastapi 做前端界面，但是有一个小问题

我使用 pyinstaller 打包，加上 -w 参数似乎fastapi 后台服务不能正常启动
```
pyinstaller -D -w -i favicon.ico main.py
```

**Comment by spring4869** [comment]
作者你好，我曾经在今年2月份的时候下载了你当时发布的yanx，当时是可以成功使用的。但是我后来不小心把软件卸载了，之后8月份又重新下载了release版本，结果在使用GUI的时候发现程序控制台在连接到127.0.0.1之后就卡住了，不能继续运行。出现这个问题的原因就是你上面提到的这个问题吗？如果是的话，我如何才能使用正常的版本呢？需要我自己修改什么配置参数吗？

**Reply by xx025** [reply]
目前我也不清楚问题出在哪儿，你可以尝试下载下来源码来运行

https://github.com/xx025/yanx#调试与开发

**Reply by xx025** [reply]
@spring4869 

不知道还是否需要，现在打包了新的版本，更换了 GUI 框架，之前的问题应该已经随之解决了


https://github.com/xx025/yanx/releases

**Reply by spring4869** [reply]
暂时不需要了，感谢作者维护
